src/main.rs

```diff
@@ -40,6 +41,8 @@ enum Cmd {
     Invoke(invoke::Cmd),
     /// Inspect a WASM file listing contract functions, meta, etc
     Inspect(inspect::Cmd),
+    /// Optimize a WASM file
+    Optimize(opt::Cmd),
```

Comment:
Maybe this command should be named `opt` rather than `optimize`?

Reply:
I like optimize better